lib/new_relic/agent/instrumentation/async_http.rb

```diff
@@ -0,0 +1,31 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'async_http/instrumentation'
+require_relative 'async_http/chain'
+require_relative 'async_http/prepend'
+
+DependencyDetection.defer do
+  named :'async_http'
```

Comment:
```suggestion
  named :async_http
```

Reply:
This one is marked as resolved without a change. Did you decide on keeping the quotes?

Reply:
oh weird, no thats my bad, sorry about that! I meant to hit commit suggestion but i accidentally hit resolve. I did that a couple times this morning when i was sleepy and thought i had fixed them all but must have missed this one.